tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -38,12 +36,9 @@ describe(__filename, () => {
     store = dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX }).store,
     ...props
   } = {}) {
-    const errorHandler = createStubErrorHandler();
-
-    store.dispatch(searchStart({ errorHandlerId: errorHandler.id, filters }));
```

Comment:
🤔

Reply:
This code doesn't really do anything. `SearchFilters` gets its `filters` prop directly as a prop, not from `mapStateToProps`, so there is no need for this.